### ISSUE TITLE
activating then deactivating an environment changes order in PATH on OSX

### ISSUE THREAD

**Issue by tdrobbin** [conda/conda#3056]
when activating an environment then deactivating it, the order of paths in PATH variable changes on OSX 10.11.5. In particular the conda path moves from the end to the beginning:

`Tylers-MacBook-Pro:lib tylerdrobbin$ echo $PATH`
`/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/tylerdrobbin/anaconda/bin`

`Tylers-MacBook-Pro:lib tylerdrobbin$ source activate main`
`discarding /Users/tylerdrobbin/anaconda/bin from PATH
prepending /Users/tylerdrobbin/anaconda/envs/main/bin to PATH`

`(main)Tylers-MacBook-Pro:lib tylerdrobbin$ echo $PATH`
`/Users/tylerdrobbin/anaconda/envs/main/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin`

`(main)Tylers-MacBook-Pro:lib tylerdrobbin$ source deactivate`
`discarding /Users/tylerdrobbin/anaconda/envs/main/bin from PATH`

`Tylers-MacBook-Pro:lib tylerdrobbin$ echo $PATH`
`/Users/tylerdrobbin/anaconda/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin`

**Comment by HugoTian** [comment]
Can you show me the conda info.  I have conda 4.1.8 and it works fine on mac, the order does not change

**Comment by tdrobbin** [comment]
Apologies - when I first noticed this behavior I updated to latest conda in
a new window, but was looking at a previous version of conda in the old
window (which is where I noticed the behavior). I thought I was using the
newer version but was still using the older! After double checking with
version 4.1.8, the path order is restored correctly after deactivating an
environment. Sorry for the simple oversight on my part. Thank you for all
your hard work!
